### PR TITLE
Add icons for private and disabled boost in web UI

### PR DESCRIPTION
**Pull request by Gargron**
New custom icons.

grafik

___

Fixes MAS-200

### CODE REVIEW

**Comment by shleeable** [comment]
Could that lock be any smaller?
chandler-freak-CMS

File: app/javascript/mastodon/components/status_action_bar.jsx [hunk 1]
```diff
@@ -360,15 +363,20 @@ class StatusActionBar extends ImmutablePureComponent {
 
     const reblogPrivate = status.getIn(['account', 'id']) === me && status.get('visibility') === 'private';
 
-    let reblogTitle = '';
+    let reblogTitle, reblogIconComponent;
+
     if (status.get('reblogged')) {
       reblogTitle = intl.formatMessage(messages.cancel_reblog_private);
+      reblogIconComponent = RepeatIcon;
```
Comment (ClearlyClaire) [inline app/javascript/mastodon/components/status_action_bar.jsx:370]: It should still be `RepeatPrivateIcon` when active depending on `publicStatus`.

File: app/javascript/mastodon/features/status/components/action_bar.jsx [hunk 2]
```diff
@@ -290,21 +293,26 @@ class ActionBar extends PureComponent {
 
     const reblogPrivate = status.getIn(['account', 'id']) === me && status.get('visibility') === 'private';
 
-    let reblogTitle;
+    let reblogTitle, reblogIconComponent;
+
     if (status.get('reblogged')) {
       reblogTitle = intl.formatMessage(messages.cancel_reblog_private);
+      reblogIconComponent = RepeatIcon;
```
Comment (ClearlyClaire) [inline app/javascript/mastodon/features/status/components/action_bar.jsx:300]: It should still be `RepeatPrivateIcon` when active depending on `publicStatus`.

File: app/javascript/mastodon/components/status_action_bar.jsx (hunk 1 above)
Comment (ClearlyClaire) [reply app/javascript/mastodon/components/status_action_bar.jsx:370]:
```suggestion
      reblogIconComponent = publicStatus ? RepeatIcon : RepeatPrivateIcon;
```

File: app/javascript/mastodon/features/status/components/action_bar.jsx (hunk 2 above)
Comment (ClearlyClaire) [reply app/javascript/mastodon/features/status/components/action_bar.jsx:300]:
```suggestion
      reblogIconComponent = publicStatus ? RepeatIcon : RepeatPrivateIcon;
```

**Review by ClearlyClaire — APPROVED** [review]

**Pull request merged**